### PR TITLE
Show in dialog box, details regarding the Github commits (#26)

### LINKED ISSUE #26: Show in dialog box, details regarding the Github commits

### PR DESCRIPTION
**Pull request by damdiana**

### CODE REVIEW

**Comment by damdiana** [comment]
Details regarding the commits:

commit message (as a commit title)
author
commit date

File: src/GithubApi.ts [hunk 1]
```diff
@@ -29,6 +29,18 @@ type GithubIssue = {
     state: string;
 }
 
+type GithubCommits = {
+
```
Comment (iampava) [inline src/GithubApi.ts]: Please remove the newline...

File: src/app.ts [hunk 2]
```diff
@@ -21,6 +21,7 @@ let dialogContent = dialog.querySelector('.details-dialog__content__dynamic');
 let dinamicContent = document.querySelector('#dinamicContent');
 let skeletonScreen = document.querySelector('.skeleton-screen');
 
+
```
Comment (iampava) [inline src/app.ts]: What is this?

File: src/app.ts [hunk 3]
```diff
@@ -265,4 +271,102 @@ async function fetchAndRenderIssues(dialog: HTMLDialogElement, repoOwner: string
...
+function displayCommits(commits: GithubCommits[], htmlElement: HTMLElement) {
+    htmlElement.innerHTML = "";
+    let ul = document.createElement('ul');
+    let descendingCommitsByDate = [...commits].sort((a, b) => {
+        let timestampA = new Date(a.commit.author.date).getTime();
+        let timestampB = new Date(b.commit.author.date).getTime();
+        return timestampB - timestampA;
+    })
+
+    commits.forEach(commit => {
```
Comment (iampava) [inline src/app.ts]: You should use the sorted commits Array: `descendingCommitsByDate`

File: src/style/style.css [hunk 4]
```diff
@@ -157,4 +157,31 @@ ul {
...
+    border-radius: 0.6em;
+}
+
+.btn-outline-left {
+    border-bottom: 3px solid grey;
+}
+
+.btn-outline-right {
+    border-bottom: 3px solid grey;
+}
```
Comment (iampava) [inline src/style/style.css]: What is this?

**Review by iampava — CHANGES_REQUESTED** [review]

File: src/style/buttons.css [hunk 5]
```diff
@@ -15,4 +15,8 @@
     padding: 0.3em 0.6em;
     border-radius: 0 0.3rem 0.3rem 0;
     cursor: pointer;
+}
+
+.btn--bottom-outline {
```
Comment (iampava) [inline src/style/buttons.css]: THis should be on the `tab` class, not a special kind of butotn.

**Review by iampava — CHANGES_REQUESTED** [review]

**Review by iampava — APPROVED** [review]

**Pull request merged**